paywall-app/package.json

```diff
@@ -6,7 +6,6 @@
     "build": "yarn build:cdn && yarn build:next",
     "build:next": "next build",
     "build:cdn": "yarn vite build",
-    "deploy": "yarn run build && next export -o out",
```

Comment:
I think we do use this command in a few places. Did you check there is no yarn deploy command?

Reply:
I will remove the fleek stuff...